### DISCUSSION TITLE
Vevői alkalmazásra vonatkozó követelmények

### DISCUSSION THREAD

**Discussion by wlln22** [nav-gov-hu/eRECEIPT#41] [Kérdések és válaszok]
1. A vevői alkalmazás támogatja.. (EU) 2015/2366 európai parlamenti és tanácsi irányelvben (PSD2 irányelv) meghatározott legalább egy API kezelését. Ez lehet AISP vagy PISP interfész is?

2. A vevői app-ban lévő adatok értéknövelt szolgáltatás céljából felhasználhatóak az ügyfél beleegyezése esetén (consent)? 
Amennyiben van erre lehetőség, van megkötés arra vonatkozólag, hogy milyen adatokat tud elemezni az app?

**Comment by BatariZ** [comment]
Az időközben megjelent rendelet (A nemzetgazdasági miniszter 8/2025. (III. 31.) NGM rendelete az e-pénztárgépek forgalmazásáról, üzemeltetéséről, valamint az e-pénztárgépek és az e-nyugta kiállításának követelményeiről), illetve az aktualizált Fejlesztői dokumentáció tartalmazza a válaszokat a feltett kérdésekre.
